storagenode/notifications/endpoint.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package notifications
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/pkg/pb"
+)
+
+var (
+	mon = monkit.Package()
+)
+
+type Endpoint struct {
+	log *zap.Logger
+}
+
+func NewEndpoint(log *zap.Logger) *Endpoint {
+	return &Endpoint{log: log}
+}
+
+func (endpoint *Endpoint) Notify(ctx context.Context, notification *pb.Notification) (_ *pb.NotifyResponse, err error) {
+	defer mon.Task()(&ctx)(&err)
+	endpoint.log.Sugar().Debug("notification", notification)
```

Comment:
The protections against random messages is missing.
The logging is at the wrong level and scope.

Reply:
that just to test we have storagenode notification system that will handle it

Reply:
Mark these as `TODO: replace with XYZ`, that way it's clearer on what is test code and what is not.